test/models-admin/Pet.js

```diff
@@ -3,10 +3,12 @@
  */
 
 module.exports = {
+    primaryKey:'id',
     identity: 'pet',
-    connection: 'flat',
+    datastore: 'flat',
     attributes: {
-        name: 'string',
-        breed: 'string'
+        id: { type: 'number', autoMigrations: { autoIncrement: true, columnType:'integer' } },
```

Comment:
It looks like logic in helper process this values, better to have tests for that as well

Reply:
Hi, 
Actually found out that this is wrong syntax
{ type: 'number', autoMigrations: { autoIncrement: true, columnType:'integer' } },
Should be
{ type: 'number',  autoIncrement: true, autoMigrations: { columnType:'integer' } },

Also when using sails-postgresql one have to create the models with the waterline 0.1.13-9 after orm initialization and call adapter.define. Have written some code for this. Please have a look. Could be good to mention this insight in the readme at leas.

https://github.com/mickelindahl/waterline-table

Reply:
Aha, I got confused. Now I understand. Yes there should be test for helper.js. Actually this is what I moved to https://github.com/mickelindahl/waterline-table just for sake of separation.I would suggest to add the onPost initialization async callback that takes (collections, datastores) asparameters. Then use this callback to setup your own model definitions when needed.

Reply:
By setting model definitions I mean calling adapter.define(...), that is creating tables for at leas  sails-postgresql

Reply:
I will check update code taking into account your comments and submit a new version